### PR TITLE
DivlessWrapper allows us to remove the unneeded 'div' wrapper

### PR DESCRIPTION
**Pull request by mtho11**
These High Order Components are very powerful for templates and such, as they can wrap (or decorate) their children. This is a simple one but good for starters and highly useful.
I didn't globally replace all the top level divs; I figured the individual authors can do this. More of these coming...

I also didn't create a jira for this since it is just a refactoring.

### CODE REVIEW

File: src/components/Nav/Navigation.tsx [hunk 1]
```diff
@@ -36,7 +37,7 @@ class Navigation extends React.Component {
 
   render() {
     return (
-      <div>
+      <DivlessWrapper>
```
Comment (josejulio) [inline src/components/Nav/Navigation.tsx]: Won’t <> and </> do the same?

Comment (josejulio) [reply src/components/Nav/Navigation.tsx]: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html

Comment (mtho11) [reply src/components/Nav/Navigation.tsx]: Hmm. I didn't know about <> and </>

Comment (jshaughn) [reply src/components/Nav/Navigation.tsx]: fwiw, <DivlessWrapper> looks less like a mistake than </>

Comment (josejulio) [reply src/components/Nav/Navigation.tsx]: Seems that `<> stuff</> ` is just a shorter version of  `<React.Fragment>stuff<React.Fragment/>`

https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html#jsx-fragment-syntax

**Pull request closed without merging**